caffe2/python/__init__.py

```diff
@@ -37,12 +37,14 @@
         cuda_path_var = 'CUDA_PATH_V' + cuda_version_1
         default_path = 'C:\\Program Files\\NVIDIA GPU Computing Toolkit\\CUDA\\v' + cuda_version
         cuda_path = os.path.join(os.getenv(cuda_path_var, default_path), 'bin')
+        cuda_extras_path = os.path.join(os.getenv(cuda_path_var, default_path), 'extras', 'CUPTI', 'lib64')
```

Comment:
Is that a correct path on Windows platform?

Reply:
(discussed)